templates/zerver/integrations/jira-plugin.md

```diff
@@ -0,0 +1,50 @@
+_If you are running JIRA version 5.2 or greater, or using the hosted JIRA provided by Atlassian, we recommend using
+the [web-hook method](#jira) above instead. This plugin supports older versions of JIRA._
+
+First, create the stream you'd like to use for JIRA notifications, and subscribe all interested parties to this
+stream. We recommend the name `jira`.
+
```

Comment:
I guess since this is the only place we could have used `create-stream.md` because of the way it is phrased we should leave it out here as well.